### DISCUSSION TITLE
Laragon Exception: "The service cannot be started..." after enabling SSL

### DISCUSSION THREAD

**Discussion by BonfireAtNight** [leokhoa/laragon#505] [Q&A]
Hey guys,

I've install Laragon on my new desktop computer today (Windows 11). Unfortunately, I've encountered an issue when I tried to enable SSL for Apache:
```
Oh no, Laragon detected exceptions!
The service cannot be started, either because it is disabled or because it has no enabled devices associated with it
```
I've googled the error message, which indicates it's a general message issued in numerous different scenarios and in regard to many applications. So I was hoping someone could point me in the direction to narrow down what might cause the problem.

I know that you can get an overview of all running services by running `services.msc`, but I don't know what services might be relevant (i.e. worth enabling/disabling for troubleshooting purposes). This morning I also read that a blocked port might also be responsible.

I would be thankful for every input!

**Comment by MaximeBoulenger** [comment]
Hey, I have the same problem. Did you find the fix?

**Reply by BonfireAtNight** [reply]
I kind of did, yes. In my own case it was because of a [Windows 11 tweak tool](https://github.com/ChrisTitusTech/winutil) I used in which I disabled "unnecessary" services. After rolling this back, things worked fine.

**Reply by MaximeBoulenger** [reply]
Thanks, I will try to restore my settings. Indeed, I disabled unnecessary services.
